Expensify/App
# feat: HTML to Markdown parser integration

TODO merge this https://github.com/Expensify/expensify-common/pull/381 and then edit commit hash
### Details

A small step towards migration to markdown and deprecation of `text`

### Fixed Issues

Fixes #2847 

### Tests


### QA Steps

1. Navigate to a conversation
2. Send a message with the following text
```
Hello 
Space
Test
```
3. Edit the comment.

The structure of the message should remain the same i.e. with new lines

### Tested On


### Screenshots


#### Web



https://user-images.githubusercontent.com/32242596/123450804-baf39d00-d5fa-11eb-800d-325b79367b69.mp4



#### Mobile Web



https://user-images.githubusercontent.com/32242596/123450821-be872400-d5fa-11eb-9a74-920d02998c0e.mp4



#### Desktop



https://user-images.githubusercontent.com/32242596/123460217-ebd8cf80-d604-11eb-9fc8-d2dea6bbdcba.mov



#### iOS



https://user-images.githubusercontent.com/32242596/123459843-7240e180-d604-11eb-9cbc-f21ef6bb3eb6.mov




#### Android



https://user-images.githubusercontent.com/32242596/123453446-335b5d80-d5fd-11eb-9d67-7c27898367a5.mp4
